api/java/org/openyolo/api/CredentialClient.java

```diff
@@ -332,6 +364,14 @@ public CredentialSaveResult getCredentialSaveResult(Intent resultData) {
         }
     }
 
+    /**
+     * Disables automatically signing a user until a successful response is given to
+     * {@link CredentialClient#getCredentialRetrieveResult(Intent)}.
+     */
+    public void disableAutoSignIn() {
```

Comment:
Using the two different terminologies (requireUserMediation vs disableAutoSignIn) is a little confusing. I don't have a good answer for this, other than to stick with `requireUserMediation`, even here, for consistency even though the usage is different. WDYT?